source/virtualBuffers/__init__.py

```diff
@@ -109,14 +109,12 @@ def obj(self):
 
 	@property
 	def label(self):
-		if self.itemType == "landmark":
-			attrs = self.textInfo._getControlFieldAttribs(self.vbufFieldIdentifier[0], self.vbufFieldIdentifier[1])
-			name = attrs.get("name", "")
-			if name:
-				name += " "
-			return name + aria.landmarkRoles[attrs["landmark"]]
-		else:
-			return super(VirtualBufferQuickNavItem,self).label
+		value = super(VirtualBufferQuickNavItem,self).label

```

Comment:
This always gets executed, but it's only used for "heading". But we also fetch the text of the TextInfo in `_getLabelForProperties`, which results in a wasted call (this property isn't cached). So, I think this call to super should be moved inside the `if` block.